src/platform-implementation-js/lib/logger.ts

```diff
@@ -272,7 +273,7 @@ function _extensionLoggerSetup(
 
   _extensionLoaderVersion = loaderVersion;
   _extensionImplVersion = implVersion;
-  _extensionUseEventTracking = opts.eventTracking;
+  _extensionUseEventTracking = opts.eventTracking ?? false;
```

Comment:
Following my previous comment that PiOpts.eventTracking shouldn't be optional, this change should be unnecessary too.